### PR TITLE
Add github-action-benchmark

### LINKED ISSUE #343: Introduce github-action-benchmark
**Description**:

Introduce [github-action-benchmark](https://github.com/benchmark-action/github-action-benchmark).

We introduced [github-action-benchmark](https://github.com/benchmark-action/github-action-benchmark) in https://github.com/yorkie-team/yorkie-js-sdk/pull/291 of JS SDK.

Let's introduce it to this repository as well.

**Why**:

- Contributors can quickly check the benchmark in PRs.

### PR DESCRIPTION
**Pull request by sinkyoungdeok**
**What this PR does / why we need it**:

we can github-action-benchmark

**Which issue(s) this PR fixes**:

Fixes #343 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Comment by hackerwins** [comment]
Could you sign the CLA?

**Comment by sinkyoungdeok** [comment]
@hackerwins 
sign completed

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,6 +39,23 @@ jobs:
...
+    - name: Store benchmark result
+      uses: benchmark-action/github-action-benchmark@v1
+      with:
+        name: Go Benchmark
+        tool: 'go'
+        output-file-path: output.txt
+        external-data-json-path: ./cache/benchmark-data.json
+        fail-on-alert: false
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        comment-on-alert: true
```
Comment (hackerwins) [inline .github/workflows/ci.yml:57]:
```suggestion
        comment-always: true
```

**Comment by hackerwins** [comment]
Forked PRs do not have comment permission, so if for commenting only on unforked PRs, we need to add the following condition:

`if: ${{ github.event.pull_request.head.repo.full_name == 'org/repo' }}`

- https://github.com/yorkie-team/yorkie-js-sdk/blob/main/.github/workflows/ci.yml#L48
- https://github.community/t/how-to-detect-a-pull-request-from-a-fork/18363

**Comment by sinkyoungdeok** [comment]
@hackerwins 

Added. thank you

**Review by hackerwins — APPROVED** [review]
LGTM. 👍

**Pull request merged**